### PR TITLE
[Documentation] Update the learning guide

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

Update the learning guide used for onboarding new internal & external contributors.

## Issue

NA

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Major breaking change
- [x] Documentation
- [ ] Other 

## List of changes

- Add direct links to the docs describing each of the V1 modules
- Link to Otto's V0 guide to Pocket
- Link to the V0 Claim & Proof lifecycle
- Add a link to [@https://twitter.com/ittaia]'s Decentralized Thoughts newsletter
- Add link to a good and simple blog on kademlia

## Testing

NA

## Required Checklist

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist
- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: docs/learning/README.md [hunk 1]
```diff
@@ -29,22 +30,38 @@ This is a general set of steps we have found to help new core team members onboa
...
-5. Over time, **and not all at once**, you can start making your way through and updating the [V1 specifications](https://github.com/pokt-network/pocket-network-protocol). _(30 hours)_
+6. **Optional**: If you’re interested, you can view the [V0 Claim & Proof Lifecycle](https://github.com/pokt-network/pocket-core/blob/staging/doc/specs/reward_protocol.md) specifications. _(2-4 hours)_
+
+   - It is a proprietary algorithm that you are unlikely to ever need to touch/modify, but it is fun & interesting to reason about and understand.
 
+7. **Eventually**: Over time, **and not all at once**, you can start making your way through and updating the [V1 specifications](https://github.com/pokt-network/pocket-network-protocol). _(15-30 hours)_
+
+   - Treat these specifications as guidelines and not sources of truth. If you are reading this, you will likely modify them at some point.
    - Mastering these concepts won't be easy but will make you an expert on Pocket V1
    - _TIP: We hope to publish it with V1 benchmarks on arxiv one day, so this is your change to contribute 🙂_
```
Comment (bryanchriswhite) [inline docs/learning/README.md:66]: Unrelated to your changes, but noticed and within reach:

```suggestion
   - _TIP: We hope to publish it with V1 benchmarks on arxiv one day, so this is your chance to contribute 🙂_
```

**Review by bryanchriswhite — COMMENTED** [review]
I noticed a small typo but otherwise this looks great and I'm looking forward to reading it - thanks @Olshansk, just in time! :smile:

**Comment by Olshansk** [comment]
Thanks for the quick review @bryanchriswhite! If you can think of any ways to improve/clarify it as you make your way through the doc, don't hesitate to change it.

**Review by bryanchriswhite — APPROVED** [review]
:raised_hands:

**Pull request merged**